CONTRIBUTING.md

```diff
@@ -0,0 +1,111 @@
+# Contributing to PowerShell Koans (PSKoans)
+
+Welcome to PSKoans and thank you for your interest in contributing!
+
+There are many ways that you can contribute, beyond writing or coding. The goal of this document is to provide a high-level overview of how you can get involved and how to interact.
+
+## PSKoans Development Tools
+
+If you want to contribute, at the very least, you'll need: 
+- [ ] [A GitHub Account](http://github.com)
+
+Here are a list of programs that should make contributing easier:
+- [ ] [GitHub Desktop](https://desktop.github.com)
+- [ ] [Visual Studio Code](https://code.visualstudio.com)
+
+Some useful VSCode extensions are as follows:
+- [ ] [PowerShell Extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=ms-vscode.PowerShell)
+- [ ] [Code Spell Checker Extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker)
```

Comment:
Is there a benefit to having both the PowerShell and [Powershell Preview](https://marketplace.visualstudio.com/items?itemName=ms-vscode.PowerShell-Preview) extensions on at the same time? or more for just giving exposure for the preview version?

The documentation on the preview extension says that if both are installed, one of them needs to be disabled.

Reply:
Yeah, it's definitely an either/or, and the preview extension is unfortunately not stable on non-Windows machines.

But when you can use it, it's much more useful, not least because PSReadLine actually works in the preview version's console. :)